liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -865,38 +865,53 @@ private Plugin getLibertyPluging() {
     private Xpp3Dom getLibertyPluginConfig() {
         // get the Liberty plugin configuration from the pom and overrides looseApplication to true.
         // ignores the plugin execution configuration just like CLI invocation (e.g. mvn liberty:create) ignoring execution configuration.
-        Plugin libertyPlugin = getLibertyPluging();
+        Plugin libertyPlugin = getLibertyPlugin();
         Xpp3Dom overrides = configuration(element(name("looseApplication"), "true"));
         return getPluginConfig(libertyPlugin, overrides);
     }
 
     private Xpp3Dom getPluginConfig(Plugin plugin, Xpp3Dom overrides) {
         Xpp3Dom config = (overrides != null) ? overrides : configuration();
-        if (plugin.getConfiguration() == null) {
-            return config;
-        } else {
-            return Xpp3Dom.mergeXpp3Dom(config, (Xpp3Dom)plugin.getConfiguration());
+        Xpp3Dom pluginConfig = (Xpp3Dom)plugin.getConfiguration();
+        if (pluginConfig != null) {
+            config = Xpp3Dom.mergeXpp3Dom(config, pluginConfig);
         }
+        return config;
     }
 
-    private final ArrayList<String> commonParams = new ArrayList<>(Arrays.asList(
+    private static final ArrayList<String> commonParams = new ArrayList<>(Arrays.asList(
             "installDirectory", "runtimeArchive", "runtimeArtifact", "libertyRuntimeVersion",
             "install", "licenseArtifact", "serverName", "userDirectory", "outputDirectory",
             "runtimeInstallDirectory", "refresh", "skip"
             ));
     
-    private final ArrayList<String> commonServerParams = new ArrayList<>(Arrays.asList(
+    private static final ArrayList<String> commonServerParams = new ArrayList<>(Arrays.asList(
             "serverXmlFile", "configDirectory", "bootstrapProperties", "bootstrapPropertiesFile",
             "jvmOptions", "jvmOptionsFile", "serverEnvFile"));
-    private ArrayList<String> createParams = new ArrayList<>(Arrays.asList(
-            "template", "libertySettingsFolder", "noPassword"
-            ));
     
-    private ArrayList<String> deployParams = new ArrayList<>(Arrays.asList(
-            "appsDirectory", "stripVersion", "deployPackages", "looseApplication", "timeout"
-            ));
+    private static ArrayList<String> createParams;
+    static {
+        createParams = new ArrayList<>(Arrays.asList(
+                "template", "libertySettingsFolder", "noPassword"
+                ));
+        createParams.addAll(commonParams);
+        createParams.addAll(commonServerParams);
+    }
+    
+    private static ArrayList<String> deployParams;
+    {
```

Comment:
missing `static` keyword for this block and the next one. I think you should define the three static variables first and then just have one static block that initializes them.